includes/definitions.inc.php

```diff
@@ -1003,6 +1003,17 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_frequency';
 $config['os'][$os]['over'][2]['text']  = 'Frequencies';
 
+$os = 'eatonups';
+$config['os'][$os]['text']             = 'Eaton UPS';
+$config['os'][$os]['type']             = 'power';
+$config['os'][$os]['icon']             = 'eaton';
```

Comment:
eaton.png exists, if Eaton UPS have a specific logo we can use it but looks ok as is.